ci-operator/step-registry/cucushift/hypershift-extended/health-check/cucushift-hypershift-extended-health-check-commands.sh

```diff
@@ -129,7 +129,10 @@ fi
 echo "check mgmt cluster's HyperShift part"
 if test -s "${SHARED_DIR}/mgmt_kubeconfig" ; then
   export KUBECONFIG=${SHARED_DIR}/mgmt_kubeconfig
-  print_clusterversion
+  # Print clusterversion only if the management cluster is an OpenShift cluster
+  if oc get ns openshift; then
+      print_clusterversion
+  fi
```

Comment:
According to the current rules, `mgmt_kubeconfig` must be an OpenShift cluster, so we do not need to add this check.

Reply:
Could you help to share the failed link?